tests/docker/rippled.Dockerfile

```diff
@@ -1,25 +1,25 @@
-FROM node:10-stretch
```

Comment:
I think commenting out this file is unnecessary; as it's invocation by docker-compose is already commented out above